stock_picking_mgmt_weight/models/stock_move.py

```diff
@@ -0,0 +1,20 @@
+# © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3.0 (https://www.gnu.org/licenses/lgpl-3.0.html)
+
+from odoo import models, fields
+
+
+class StockMove(models.Model):
+    _inherit = 'stock.move'
+
+    theoretical_qty = fields.Float(string='Theoretical quantity demanded')
+    tare = fields.Float(string='Tare')
+    date_tare_weight = fields.Date(string='Tare weight date')
```

Comment:
More coherent naming:
```suggestion
    date_tare = fields.Date(string='Tare date')
```